css/style.css

```diff
@@ -0,0 +1,134 @@
+.body {
+  margin: 0 auto;
+  padding-left: 5px;
+}
+.first-field {
+  display: flex;
+  flex-wrap: wrap;
+}
+.svg-ff {
+  padding: 1px;
+}
+.kiwi-tag {
+  fill: orange;
+}
+.lake {
+  fill: blue;
+}
+.kiwi-object {
+  width: 150px;
+}
+.kiwi-base64 {
+  width: 150px;
+}
+/*for 3*/
+.circle-anim {
+  animation-duration: 2s;
+  animation-name: rotate-loading-circle;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+@keyframes rotate-loading-circle {
+  from {
+    transform: rotate(0deg);
+  }
+  to { 
+    transform: rotate(360deg);
+  }
+}
+/*for 4*/
+@media all and (min-width: 1800px) {
+  .svg-resp {
+    width: 800px;
+    height: 400px;
+  }
+}
+@media all and (max-width: 1800px) and (min-width: 1201px)  {
+  .svg-resp {
+    width: 600px;
+    height: 300px;
+  }
+}
+@media all and (max-width: 1200px) and (min-width: 961px)  {
+  .svg-resp {
+    width: 500px;
+    height: 250px;
+  }
+}
+@media all and (max-width: 960px) and (min-width: 769px)  {
+  .svg-resp {
+    width: 400px;
+    height: 200px;
+  }
+}
+/*for 5*/
+.container {
+  height: 100%;
+  position: relative;
+}
+.container__box {
+  margin: 0;
+  padding: 20px;
+  text-align: center;
+  color: #ffffff;
+  background: #9e54bd;
+  position: relative;
+  z-index: 5;
```

Comment:
Why do you need `z-index` here?

Reply:
It is necessary for correct work of animation. I edited to 1.

Reply:
If z-index missed, default value - auto.